src/make_data/banner_intervention/README.md

```diff
@@ -0,0 +1,137 @@
+ want to visit the Start a Business checker.
+- The banner is tracked on Google Analytics as an EVENT hit:
+  - When users are shown the banner: `eventCategory = 'interventionBanner'`
+                                     `eventAction = 'interventionShown'`
+  - When users select the banner:    `eventCategory = 'interventionBanner'`
+                                     `eventAction = 'interventionClicked'`
+  - When users reject the banner:    `eventCategory = 'interventionBanner'`
+                                     `eventAction = 'interventionDismissed'`
+
+###### banner_selects_results_page.sql ######
+- This query extracts the proportion of users that access the results page once
+  selecting the SaB checker, i.e. total number of users that access the results
+  page / total number of users that access the SaB checker via the banner
+
+###### banner_selects_one_result.sql ######
+- This query extracts the proportion of users that select at least one link on the
+  results page once finishing the SaB checker, i.e. total number of users that
+  select 1 link on the results page / total number of users that access the SaB
+  checker via the banner
+
+###### banner_select_reject_ignore.sql ######
+- This query extracts the proportion of users that select the checker via the
+  banner intervention, reject the checker via the banner intervention, and that
+  ignore the checker via the banner intervention, i.e. total number of users that
+  select the checker / total number of users that are shown the banner intervention;
+  total number of users that reject the checker / total number of users that are
+  shown the banner intervention; total number of users that ignore the checker /
+  total number of users that are shown the banner intervention;
+
+## Assumptions and caveats
+
+#### Nonequivalent group (email)
+
+- This group is defined by users that access the pagePath that starts with:
+  `/next-steps-for-your-business?crn`. The query string `crn` represents users
+  that have landed on the pagePath from the Companies House email.
+
+###### email_selects_one_result.sql ######
+
+- A REGEXP_CONTAINS statement defines all results pagePaths as a string that starts
+  with the below, and matches any character, any number of times.
+- These are the pagePaths included that are linked from the results page:
+`/run-business-from-home`
+`/renting-business-property-tenant-responsibilities`
+`/browse/business/premises-rates`
+`/running-a-limited-company/signs-stationery-and-promotional-material`
+`/business-finance-support?business_stages%5B%5D=not-yet-trading`
+`/business-coronavirus-support-finder`
+`/export-goods`
+`/research-export-markets`
+`/import-goods-into-uk`
+`/vat-registration`
+`/vat-registration/when-to-register`
+`/get-ready-to-employ-someone`
+`/browse/employing-people`
+`/corporation-tax`
+`/licence-finder`
+`https://www.abi.org.uk/products-and-issues/choosing-the-right-insurance/business-insurance`
```

Comment:
does this show up in our tracking?

Reply:
I have been having a look, and I could change this REGEX to checking `hits.eventInfo.eventLabel` value , rather than `hits.page.pagePath`. 

This is because if a user selects a link on the result page, this is  tracked as an EVENT hit (eventCategory = 'SmartAnswerClicked' and eventLabel = the url they select). 

So, to make it more accurate, I could only count SmartAnswerClicked events and then search the eventLabel for the appropriate URL.  

This would make it more accurate, as currently, the user could actually go to one of those pagePaths during the session, before they even used the checker. Therefore they did not truly select a result link during the session. 

What do you think?  

**EXAMPLE TRACKING** 

| pagePath  | eventCategory  | eventAction  | eventLabel  | type  |
|---|---|---|---|---|
| /next-steps-for-your-business/results?...=no | SmartAnswerClicked  |  2.11 | /vat-businesses  |  EVENT |

Reply:
That sounds like a great improvement! I'm not sure we even need to know which URL they clicked for these queries as long as they click a result on the results page?

Reply:
That's correct - I checked earlier and 'SmartAnswerClicked' is the only event for those results links, so I've just changed the script to include this eventCategory rather than the list of URLs. Please see commit https://github.com/alphagov/govuk-entity-personalisation/pull/36/commits/463bb062611b2c7985004ad9e57be25f7d0aa424